src/string_util.h

```diff
@@ -70,4 +70,4 @@ using std::stoul;  // NOLINT(misc-unused-using-decls)
 
 }  // end namespace benchmark
 
-#endif  // BENCHMARK_STRING_UTIL_H_
+#endif  // BENCHMARK_STRING_UTIL_H_
```

Comment:
Please add newline back

Reply:
![image](https://github.com/google/benchmark/assets/20411128/34085215-841f-490b-a13b-62c98a6886e0)


I matched it with the main repo and the lines are same.

Reply:
If you look at that line in the `Files Changed` view here,
you'll clearly see an indicator that there is no new line at the end of the file,
and that indicator does not show up on the left/original side of the diff.

Reply:
Not done.